### DISCUSSION TITLE
Forgified Qulit?

### DISCUSSION THREAD

**Discussion by RoomAlex68** [Sinytra/Connector#119] [Ideas]
Will there be a Forgified Qulit for Qulit Mods that run on Qulit mod loader to work with Sinytra Connector to load them on Forge Mod Loader like with Fabric Mods at moment?
Will that a possibility in the future?

**Comment by RoomAlex68** [comment]
And I know that Qulit is a fork of Fabric

**Comment by digi2303** [comment]
it will be good

**Comment by RoomAlex68** [comment]
The API would have been QFAPI or FQAPI or something

**Comment by kristibektashi** [comment]
A Forgified QSL (Quilt Standard Libraries) would mean that we might finally be able to use mods from all major mod loaders alongside each other (as long as NeoForge remains fully compatible with the original Forge). I would love for that to happen, as I will finally be able to play with all the mods I like without worrying about the mod loader

**Reply by RoomAlex68** [reply]
Yeah. They should develop Forgified QSL for Qulit Mods to run connector with Forge and Fabric mods. Maybe not right now since connector is in beta. Once it's in a stable version they maybe will being a developed a Forgified QSL for Connector.

**Comment by forgolotl** [comment]
most quilt mods work on fabric anyways and qfapi is a straight copy of fapi with less compatibility so theres just no reason to do that

**Reply by kristibektashi** [reply]
QFAPI for Forge is indeed pointless, however QSL for Forge would actually be useful, as it would allow the few Quilt-exclusive mods to run with Sinatra Connector (because yes they do exist)

**Comment by Swagboi7** [comment]
> Will there be a Forgified Qulit for Qulit Mods that run on Qulit mod loader to work with Sinytra Connector to load them on Forge Mod Loader like with Fabric Mods at moment? Will that a possibility in the future?

I'll do you one better - Neoforgified Quilt because there will inevitably come Neoforge exclusives too.

Though that one depends on Forgified Fabric API being able to update past Forge 47.1.3.

**Reply by Swagboi7** [reply]
and as previously mentioned above, Sinytra being in a stable enough state to integrate Quilt.

**Reply by RoomAlex68** [reply]
Once Sinytra Connector is in a stable state then NeoForgified QSL( Qulit Standard Libraries ) will developed when NeoForge is completed because NeoForge will start to have its own exclusive mods for it and having Sinytra Connector running all four mod loaders (as long NeoForge supports Forge still) it will be a the biggest thing ever.
Someone would create a really great modpack with mods from the four mod loaders.

**Reply by RoomAlex68** [reply]
I know a Qulit exclusive mod from Qulit Mod Loader called Plants vs Zombies: Cubed that is currently on 1.19.2 but soon it will be moved to the 1.20 versions of game.
